.actrc

```diff
@@ -0,0 +1,2 @@
+# These default configurations will be applied when running act locally for GitHub Action testing
+-P ubuntu-latest=ghcr.io/catthehacker/ubuntu:act-latest
```

Comment:
When [first running `act`](https://github.com/nektos/act#first-act-run), the user is asked what default image to use. So, I see the `.actrc` file more of a user defined configuration that may change from user-to-user and not something we'd want to checkin.

Reply:
I created this file so that the user wouldn't need to decide [which image](https://github.com/nektos/act#runners) to choose, since the minimal one doesn't have the necessary packages pre-installed to operate. I'm wondering if it would be better to remove this user setting file and clarify in the documentation section instead?

Reply:
Yeah, personally, I would have a section on how to configure the `act` cli in the `CONTRIBUTING.md` under the [development](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#development) section.